ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -224,6 +204,13 @@ struct ZipBuilder {
     let carthageCoreDiagnostics: URL = createCarthageCoreDiagnostics(fromPods: installedPods,
                                                                      inProjectDir: projectDir)
 
+    do {
+      let coreDiagnosticModuleMap = frameworks["FirebaseCoreDiagnostics"]![0].appendingPathComponent("Modules")
```

Comment:
Instead of force unwrapping (which would likely give a not-very-helpful error) could we `guard let ` and handle the optional case? Also `first` is preferred over `[0]` since it'll fail gracefully.